autoscale/src/main/java/com/sequenceiq/periscope/service/UsageReportingService.java

```diff
@@ -93,4 +96,15 @@ private UsageProto.AutoscaleScalingStatusType.Value getAutoscaleStatusType(Scali
                 return UsageProto.AutoscaleScalingStatusType.Value.UNSET;
         }
     }
+
+    private UsageProto.AutoscaleScalingAdjustmentType.Value getAutoScalingAdjustmentType(ScalingAdjustmentType scalingAdjustmentType) {
+        switch (scalingAdjustmentType) {
+            case STOPSTART:
+                return UsageProto.AutoscaleScalingAdjustmentType.Value.STOPSTART;
```

Comment:
I can see `usage.proto` was updated / synced, but cannot find the respective generated Java class included in this commit. How is this supposed to work?

Reply:
I have updated the usage.proto again to match it with the one in thunderhead as it was updated so in the autogenerated file there are many more changes as well.